chain/evm/compass.go

```diff
@@ -608,6 +608,47 @@ func (t compass) provideEvidenceForValidatorBalance(ctx context.Context, queueTy
 	return g.Return()
 }
 
+func (t compass) provideEvidenceForReferenceBlock(ctx context.Context, queueTypeName string, msgs []chain.MessageWithSignatures) error {
+	logger := liblog.WithContext(ctx).WithField("queue-type-name", queueTypeName)
+	logger.Debug("start processing reference block request")
+
+	var g whoops.Group
+	for _, msg := range msgs {
+		g.Add(
+			whoops.Try(func() {
```

Comment:
Remove the `whoops` lib usages.

Reply:
I removed it. We now return an error after failing a single message, which should be what we wanted anyway.